packages/perl_locale_gettext.rb

```diff
@@ -1,32 +1,28 @@
-
 require 'package'
 
 class Perl_locale_gettext < Package
   description 'Locale::gettext - message handling functions'
   homepage 'https://metacpan.org/pod/Locale::gettext'
-  compatibility 'all'
   license 'GPL-1+ or Artistic'
-  version '1.07-1'
+  version '1.07-2'
```

Comment:
Can we put version above license for consistency?

Reply:
That's my default for new packages. This was just a bunch of straight rebuilds with newer perl... Feel free to change that in any packages here you notice that on, as that will also save me work.

Reply:
(by default I meant that all I was changing was the version number as I was updating these packages...)

Reply:
I meant swap the lines to be consistent with all the other packages.

Reply:
No I got that... it's on my list... there are just a lot of these files which could have that issue.